qgreenland/config/datasets/earthquakes.py

```diff
@@ -1,24 +1,49 @@
 import datetime as dt
 
-from qgreenland.models.config.asset import ConfigDatasetHttpAsset
+from qgreenland.models.config.asset import ConfigDatasetCommandAsset
 from qgreenland.models.config.dataset import ConfigDataset
 
 
 query_start_date = dt.date(1900, 1, 1)
 query_end_date = dt.date(2021, 1, 1)
 
+
+_longitude_step = 2
+# The USGS site seems to be rate limited. We'll get errors (name resolution
+# error) if too much time is spent holding open connection(s) with the site.
+# Faster downloads seem to work more reliably than slower ones. Do we need to do
+# anything more, e.g. add sleeps to each `wget`, to give the server a break?
+# TODO: What error was Trey getting?
```

Comment:
@trey-stafford when you get a sec could you fill this in? I struggled to find the actual error message in Slack for some reason. :shrug: